dev/com.ibm.ws.beanvalidation.v11_fat/fat/src/com/ibm/ws/beanvalidation/fat/tests/BasicValidation10CommonTest.java

```diff
@@ -196,6 +198,7 @@ public void testCustomConstraintValidatorFactory10() throws Exception {
      */
     @Test
     public void testBuildApacheConfiguredValidatorFactory10() throws Exception {
+        assumeTrue(bvalVersion < 20);
```

Comment:
Never used `assumeTrue()` before.  If the assumption is false, does the test get reported as skipped?

Reply:
Yup. False marks the test skipped.